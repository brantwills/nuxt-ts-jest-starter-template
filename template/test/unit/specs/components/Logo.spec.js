import { shallow } from 'vue-test-utils'
import { createRenderer } from 'vue-server-renderer'
import Component from '~/components/Logo.vue'

describe(`Component Logo`, () => {

  it('renders to a snapshot', () => {
    const renderer = createRenderer()
    const wrapper = shallow(Component, {})
    renderer.renderToString(wrapper.vm, (err, str) => {
      if (err) throw new Error(err)
      expect(str).toMatchSnapshot()
    })
  })

})