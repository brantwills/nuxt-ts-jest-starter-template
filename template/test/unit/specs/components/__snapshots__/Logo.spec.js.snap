// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component Logo renders to a snapshot 1`] = `
<div class="VueToNuxtLogo">
    <div class="Triangle Triangle--two"></div>
    <div class="Triangle Triangle--one"></div>
    <div class="Triangle Triangle--three"></div>
    <div class="Triangle Triangle--four"></div>
</div>
`;
